### PR TITLE
New Parse EmailHeaders script

### PR DESCRIPTION
**Pull request by dem-evgeny**

### CODE REVIEW

**Comment by dem-evgeny** [comment]
@JonathanMeler Any news regarding it?

File: Scripts/script-ParseEmailHeaders.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  id: ParseEmailHeaders
+  version: -1
+name: ParseEmailHeaders
+script: |-
+  from email.parser import HeaderParser
+
+  ec = {"Email.Headers" : {}}
+  md = ""
+
+  # Get the file path of the email file as it lokated in Demisto server
```
Comment (JonathanMeler) [inline Scripts/script-ParseEmailHeaders.yml]: Grammar and speling mistakes

File: Scripts/script-ParseEmailHeaders.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  filePath = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryid']})[0]['Contents']['path']
```
Comment (JonathanMeler) [inline Scripts/script-ParseEmailHeaders.yml]: Use demisto.get to extract the path (could prompt an error)

File: Scripts/script-ParseEmailHeaders.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+tags:
+- email
+comment: This automation parses headers from an email file.
+enabled: true
+args:
+- name: entryid
+  required: true
+  default: true
+  description: Entry id of an email file
+scripttarget: 0
```
Comment (JonathanMeler) [inline Scripts/script-ParseEmailHeaders.yml]: I think you need to add `releaseNotes` for the test to work

Comment (dem-evgeny) [reply Scripts/script-ParseEmailHeaders.yml]: No need for a new script.

File: Scripts/script-ParseEmailHeaders.yml (hunk 1 above)
Comment (JonathanMeler) [reply Scripts/script-ParseEmailHeaders.yml]: Maybe: Get the email file location on the Demisto server

**Comment by dorsha** [comment]
@dem-evgeny I'm closing this PR since no one touched it for 24 days, please fix and re-open.

**Pull request closed without merging**